tests/XperienceCommunity.QueryExtensions.Tests/XperienceCommunity.QueryExtensions.Tests.csproj

```diff
@@ -17,7 +17,8 @@
     <PackageReference Include="FluentAssertions" Version="6.6.0" />
     <PackageReference Include="NSubstitute" Version="4.3.0" />
 	<PackageReference Include="AutoFixture.AutoNSubstitute" Version="4.17.0" />
-	<PackageReference Include="Kentico.Xperience.Libraries.Tests" Version="[13.0.0, 13.1.0)" />
```

Comment:
Older versions of `Kentico.Xperience.Libraries.Tests` were broken, so this upgrades them to the latest to support more advanced features. The library itself still supports older version of KX 13